run.sh

```diff
@@ -117,16 +117,16 @@ docker run --name client -h client --cap-add=NET_ADMIN -t siff /bin/bash -c "
     route add default gw client-router
     iptables -A OUTPUT -j NFQUEUE --queue-num 0
     iptables -A INPUT -j NFQUEUE --queue-num 1
-    /go/bin/cs4516project -mode client" >> client.log 2>&1 &
-
-docker run --name attacker -h attacker --cap-add=NET_ADMIN -t siff /bin/bash -c "
-    echo -e '$HOSTS' > /etc/hosts
-    ip addr flush dev eth0
-    ip addr add $ATTACKER dev eth0
-    route add -host legacy-router/32 eth0
-    route add -host server/32 gw legacy-router
-    route add default gw legacy-router
-    iptables -A OUTPUT -j NFQUEUE --queue-num 0
-    /go/bin/cs4516project -mode attacker" >> attacker.log 2>&1
+    /go/bin/cs4516project -mode client" >> client.log 2>&1
+
+#docker run --name attacker -h attacker --cap-add=NET_ADMIN -t siff /bin/bash -c "
```

Comment:
​That was to simplify logs and remove ambiguity in capability updates. I at
least have no clue if it works, didn't really poke around with that too
much.​

On Mon, Dec 14, 2015 at 6:45 PM, Alec Thompson <notifications@github.com>
wrote:

> In run.sh
> <https://github.com/ThomasJClark/cs4516project/pull/30#discussion_r47577514>
> :
>
> > @@ -117,16 +117,16 @@ docker run --name client -h client --cap-add=NET_ADMIN -t siff /bin/bash -c "
> >      route add default gw client-router
> >      iptables -A OUTPUT -j NFQUEUE --queue-num 0
> >      iptables -A INPUT -j NFQUEUE --queue-num 1
> > -    /go/bin/cs4516project -mode client" >> client.log 2>&1 &
> > -
> > -docker run --name attacker -h attacker --cap-add=NET_ADMIN -t siff /bin/bash -c "
> > -    echo -e '$HOSTS' > /etc/hosts
> > -    ip addr flush dev eth0
> > -    ip addr add $ATTACKER dev eth0
> > -    route add -host legacy-router/32 eth0
> > -    route add -host server/32 gw legacy-router
> > -    route add default gw legacy-router
> > -    iptables -A OUTPUT -j NFQUEUE --queue-num 0
> > -    /go/bin/cs4516project -mode attacker" >> attacker.log 2>&1
> > +    /go/bin/cs4516project -mode client" >> client.log 2>&1
> > +
> > +#docker run --name attacker -h attacker --cap-add=NET_ADMIN -t siff /bin/bash -c "
>
> Is the attacker being commented out leftover from testing, or does it not
> work and need to be updated?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ThomasJClark/cs4516project/pull/30/files#r47577514>.
>



-- 
Brett Ammeson